lib/compiler/doc/src/compile.xml

```diff
@@ -512,13 +520,23 @@ module.beam: module.erl \
           </item>
 
           <tag><c>no_line_info</c></tag>
-
           <item>
             <p>Omits line number information to produce a slightly
 	      smaller output file.
 	    </p>
           </item>
 
+          <tag><c>no_lint</c></tag>
+          <item>
+            <p>Skips the pass that checks for errors and warnings. Only
+            applicable together with the <c>from_abstr</c> option. This is
+            mainly for implementations of other languages on top of Erlang,
+            which have already done their own checks to guarantee
+            correctness of the code.</p>
+            <p>Caveat: If the code contains errors, the compiler may crash
+            instead of producing any readable error messages.</p>
```

Comment:
This part gives me the impression that it will always crash hard in the presence of errors, but at worst it could produce incorrect code without crashing. If we decide to add this option then I think it should be strongly noted that we give no guarantees.

Reply:
How about "Caveat: When this option is used, there are no guarantees that the code output by the compiler is correct and safe to use. The responsibility for correctness lies on the code or person generating the abstract format. If the code contains errors, the compiler may crash or produce unsafe code."

Reply:
Sounds good to me. :)